extensions/sample-command/build.gradle

```diff
@@ -36,4 +41,6 @@ repositories {
 	maven {
 		url "https://repository-cdn.liferay.com/nexus/content/groups/public"
 	}
-}
\ No newline at end of file
+}
+
+test.dependsOn tasks.getByPath(':cli:testClasses')
```

Comment:
shouldn't be needed right?  since `testCompile project(path: ":cli", configuration: "testArtifacts")` is going to need to build the configuration and to build that configuration it is going to build the test classes.

Reply:
Also be sure and run SF before commiting.